StandardCyborgNetworking/StandardCyborgNetworking/Operations/ServerOperationError.swift

```diff
@@ -18,7 +18,7 @@ public enum ServerOperationError: Error {
     case invalidUsernamePassword
     case invalidInput(message: String)
     
-    public var localizedDescription: String {
```

Comment:
The `LocalizedError` protocol is better suited for displaying human-readable errors which is why we transition to using that here.